java/code/src/com/suse/manager/reactor/messaging/ApplyStatesEventMessage.java

```diff
@@ -123,6 +155,15 @@ public Long getServerId() {
         return stateNames;
     }
 
+    /**
+     * Return the state specific pillar data
+     *
+     * @return the pillar data
+     */
+    public Map<String, Object> getPillar() {
```

Comment:
Can we directly return Optional here or would returning an empty map also make sense?

Reply:
done